src/platform/backends/hyperv_api/hcn/hyperv_hcn_api_table.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+// clang-format off
+#include <windows.h>
+#include <computenetwork.h>
+#include <objbase.h> // for CoTaskMemFree
+// clang-format on
+
+#include <multipass/singleton.h>
```

Comment:
Nitpick: better place own includes first. That way, they couldn't rely on something that this file includes first without including it themselves.